esp/logging/logginglib/LogFailSafe.cpp

```diff
@@ -78,7 +78,8 @@ CLogFailSafe::CLogFailSafe(IPropertyTree* cfg, const char* pszService, const cha
 
 CLogFailSafe::~CLogFailSafe()
 {
-    ESPLOG(LogMax, "CLogFailSafe::~CLogFailSafe()");
+//    ESPLOG(LogMax, "CLogFailSafe::~CLogFailSafe()");
+    LOG(LegacyMsgCatMax, "CLogFailSafe::~CLogFailSafe()");
```

Comment:
@wangkx can confirm but this seems like a good candidate for DBGLOG

Reply:
I do agree.